tests/results/DiffTest.py

```diff
@@ -17,6 +17,11 @@ def test_add_lines(self):
         self.uut.add_lines(0, ['t'])
         self.uut.add_lines(0, [])
 
+    def test_add_line(self):
```

Comment:
Oh So that was the way the test was supposed to be written.
Thanks :+1: 
I'll try it right now.